kubeflow/pachyderm/README.md

```diff
@@ -0,0 +1,40 @@
+# Pachyderm
+
+To deploy pachyderm
+
+```shell
+ks pkg install kubeflow/pachyderm@${VERSION}
+ks generate pachyderm pachyderm
+ks apply ${ENV} -c pachyderm
+```
+
+Once you've deployed pachyderm, follow pachyderm's 
+[user_guide](http://pachyderm.readthedocs.io/en/latest/getting_started/getting_started.html).
+You will need to install Pachyderm's CLI (pachctyl) and run `pachctl -n ${NAMESPACE} port-forward &`
+in order to interact with Pachyderm.
+
+**Warning** By default this local model of Pachyderm which uses ephmeral volumes
+to store data. It is not suitable for production use as data will be lost if the pods get  restarted.
```

Comment:
Is this true for the GCP deploy as well?

Reply:
No updated the doc to make this cleaner.